### PR TITLE
Sync accurate details in doc.

### PR DESCRIPTION
**Pull request by Rupesh-git-eng**
The clusters are by default set up with ETP=local, but as the patch for local is yet not exist, we still balance the traffic even with ETP set to local.
Post the patch release (4.10), we will support local.

spec:
clusterIP: 172.30.64.132
externalTrafficPolicy: Local
healthCheckNodePort: 32424

### CODE REVIEW

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ The OVN-Kubernetes Container Network Interface (CNI) cluster network provider ha
 The default value, `cluster`, is supported for both parameters.
 This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
 
+[NOTE]
+====
+Openshift Installer provisioned infrastructure (IPI) installation on Azure cloud, the installer create Loadbalancer service with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[BZ#1903408].
```
Comment (Miciah) [inline modules/nw-ovn-kuberentes-limitations.adoc:16]: It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.  

The link URL has one BZ number, and the link text has a different number.  I think you want the latter: <https://bugzilla.redhat.com/show_bug.cgi?id=1903408>.  

Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing text?

Comment (Rupesh-git-eng) [reply modules/nw-ovn-kuberentes-limitations.adoc]: Hello @Miciah 

Thank you for the comment.

> It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.

Correct the lines a bit to include just an ingress service. 
I think, we only create such LB service type in Azure, I'm not seeing ETP set in UPI. 
When I create LB type svc using oc expose for any random service, it set ETP to cluster which is correct. 

> The link URL has one BZ number, and the link text has a different number. I think you want the latter: https://bugzilla.redhat.com/show_bug.cgi?id=1903408.

Corrected. 

> Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing text?

Yes, the OVN limitation is confusing when it comes to the Ingress service that we create as part of the installation with respect to ETP. This change will just give more clarity on it.

Comment (Miciah) [reply modules/nw-ovn-kuberentes-limitations.adoc]: > I think, we only create such LB service type in Azure, I'm not seeing ETP set in UPI.

Whether the cluster uses IPI or UPI, the resulting cluster is supposed to be the same.  Anyway, the ingress operator looks at the platform type in the infrastructure config (which you can check in `oc get infrastructures.config/cluster`), so UPI or IPI shouldn't make a difference.  

We default to "LoadBalancerService" for all cloud platforms; see <https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L84-L102> and <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/pkg/operator/controller/ingress/controller.go#L359-L360>.  We set `externalTrafficPolicy` on LoadBalancer services on all platforms except for IBM Cloud and Power VS; see <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/assets/router/service-cloud.yaml#L14-L16> and <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/pkg/operator/controller/ingress/load_balancer_service.go#L401-L406>.  

> Yes, the OVN limitation is confusing when it comes to the Ingress service that we create as part of the installation with respect to ETP. This change will just give more clarity on it.

Is the problematic behavior specific to Azure?  If not, then I'm concerned that adding a note saying that the problem existed on Azure would imply that it didn't exist on other platforms and would mislead users.

Comment (Rupesh-git-eng) [reply modules/nw-ovn-kuberentes-limitations.adoc]: Thank you @Miciah, I have applied correction accordingly.

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 2]
```diff
@@ -11,6 +11,11 @@ The OVN-Kubernetes Container Network Interface (CNI) cluster network provider ha
 The default value, `cluster`, is supported for both parameters.
 This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
 
+[NOTE]
+====
+For openshift deployments in cloud platforms except IBM cloud and Power VS, the ingress operator create Loadbalancer type service for the ingress with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1903408[BZ#1903408].
```
Comment (Amrita42) [inline modules/nw-ovn-kuberentes-limitations.adoc:16]: replace **openshift** with **{product-title}** 
in **all** cloud platforms
Capitalize **Ingress** and **Operator**
Replace **create** with **creates**
Add an article **a** before `Loadbalancer type service for....`

Comment (Amrita42) [reply modules/nw-ovn-kuberentes-limitations.adoc:16]: Please squash all the five commits into one commit as per OpenShift doc guidelines.

Comment (Rupesh-git-eng) [reply modules/nw-ovn-kuberentes-limitations.adoc:16]: Hi @Amrita42 

We might not need this anymore as the engineering has agreed to change product behavior which the doc bug was going to cover. I'll close this if not needed in a day or two.

Comment (Amrita42) [reply modules/nw-ovn-kuberentes-limitations.adoc:16]: yeah sure , when closing the bug , you can mention as not a bug. Thanks.

**Pull request closed without merging**